### PR TITLE
Use auto control in standalone version

### PR DESCRIPTION
**Pull request by Pessimistress**
#### Change List
- remove `onViewportChange` hack, use the new automatic control API instead
- minor fix to test app

### CODE REVIEW

File: modules/lite/src/deckgl.js [hunk 1]
```diff
@@ -80,7 +80,9 @@ export default class DeckGL extends Deck {
         width: '100%',
         height: '100%',
         canvas: deckCanvas,
-        controller: props.controller || Controller
+        controller: props.controller || Controller,
+        onViewStateChange: ({viewState}) =>
+          this._map && this._map.setProps({viewState}) && viewState
```
Comment (ibgreen) [inline modules/lite/src/deckgl.js]: Maybe comment what returning `viewState` does? It shouldn't be needed?

File: modules/lite/test/index.js [hunk 2]
```diff
@@ -41,7 +41,7 @@ const nonGeoExample = new deck.DeckGL({
   container: document.getElementById('non-geo'),
   mapbox: false /* disable map */,
   views: [new deck.OrbitView()],
-  viewState: {distance: 1, rotationX: 45, rotationOrbit: 30, zoom: 0.05},
+  viewState: {distance: 1, fov: 50, rotationX: 45, rotationOrbit: 30, zoom: 0.05},
```
Comment (ibgreen) [inline modules/lite/test/index.js]: * Ideally projection parameters like `fov` that are not really related to the viewState should go into the `OrbitView` itself. Not 1009% sure if that works for `OrbitView`.

* Any reason to change `viewState` to `initialViewState`? Do any docs need updating in this regard?

File: modules/lite/src/deckgl.js (hunk 1 above)
Comment (Pessimistress) [reply modules/lite/src/deckgl.js]: `this._map.setProps` returns the map itself.

File: modules/lite/test/index.js (hunk 2 above)
Comment (Pessimistress) [reply modules/lite/test/index.js]: `initialViewState` is needed to enable auto control
